### ISSUE TITLE
[BUG] Docker build fails on main branch

### ISSUE THREAD

**Issue by Tockra** [paperless-ngx/paperless-ngx#1146]
### Description

I'm trying to build the a docker image with help of the given docker file.
But step 10/16 fails:
```bash
 => CACHED [compile-frontend 5/5] RUN set -eux   && ./node_modules/.bin/ng build --configuration production                                                                                                                             0.0s
 => ERROR [main-app 10/16] RUN --mount=type=bind,from=qpdf-builder,target=/qpdf     --mount=type=bind,from=psycopg2-builder,target=/psycopg2     --mount=type=bind,from=pikepdf-builder,target=/pikepdf   set -eux   && echo "Installi  3.0s
------
 > [main-app 10/16] RUN --mount=type=bind,from=qpdf-builder,target=/qpdf     --mount=type=bind,from=psycopg2-builder,target=/psycopg2     --mount=type=bind,from=pikepdf-builder,target=/pikepdf   set -eux   && echo "Installing qpdf"     && apt-get install --yes --no-install-recommends /qpdf/usr/src/qpdf/libqpdf28_*.deb     && apt-get install --yes --no-install-recommends /qpdf/usr/src/qpdf/qpdf_*.deb   && echo "Installing pikepdf and dependencies"     && python3 -m pip install --no-cache-dir /pikepdf/usr/src/pikepdf/wheels/packaging*.whl     && python3 -m pip install --no-cache-dir /pikepdf/usr/src/pikepdf/wheels/lxml*.whl     && python3 -m pip install --no-cache-dir /pikepdf/usr/src/pikepdf/wheels/Pillow*.whl     && python3 -m pip install --no-cache-dir /pikepdf/usr/src/pikepdf/wheels/pyparsing*.whl     && python3 -m pip install --no-cache-dir /pikepdf/usr/src/pikepdf/wheels/pikepdf*.whl     && python -m pip list   && echo "Installing psycopg2"     && python3 -m pip install --no-cache-dir /psycopg2/usr/src/psycopg2/wheels/psycopg2*.whl     && python -m pip list:
#27 0.653 + echo Installing qpdf
#27 0.653 Installing qpdf
#27 0.654 + apt-get install --yes --no-install-recommends /qpdf/usr/src/qpdf/libqpdf28_10.6.3-1_amd64.deb
#27 0.660 Reading package lists...
#27 0.669 Building dependency tree...
#27 0.670 Reading state information...
#27 0.680 The following NEW packages will be installed:
#27 0.680   libqpdf28
#27 0.690 0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
#27 0.690 Need to get 0 B/792 kB of archives.
#27 0.690 After this operation, 2636 kB of additional disk space will be used.
#27 0.690 Get:1 /qpdf/usr/src/qpdf/libqpdf28_10.6.3-1_amd64.deb libqpdf28 amd64 10.6.3-1 [792 kB]
#27 0.845 debconf: delaying package configuration, since apt-utils is not installed
#27 0.981 Selecting previously unselected package libqpdf28:amd64.
(Reading database ... 13013 files and directories currently installed.)
#27 0.999 Preparing to unpack .../libqpdf28_10.6.3-1_amd64.deb ...
#27 1.049 Unpacking libqpdf28:amd64 (10.6.3-1) ...
#27 1.264 Setting up libqpdf28:amd64 (10.6.3-1) ...
#27 1.330 Processing triggers for libc-bin (2.31-13+deb11u2) ...
#27 1.354 ldconfig: /usr/local/lib/libjbig2enc.so.0 is not a symbolic link
#27 1.354
#27 1.486 + apt-get install --yes --no-install-recommends /qpdf/usr/src/qpdf/qpdf_10.6.3-1_amd64.deb
#27 1.504 Reading package lists...
#27 1.532 Building dependency tree...
#27 1.534 Reading state information...
#27 1.549 The following NEW packages will be installed:
#27 1.549   qpdf
#27 1.555 0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
#27 1.555 Need to get 0 B/125 kB of archives.
#27 1.555 After this operation, 314 kB of additional disk space will be used.
#27 1.555 Get:1 /qpdf/usr/src/qpdf/qpdf_10.6.3-1_amd64.deb qpdf amd64 10.6.3-1 [125 kB]
#27 1.634 debconf: delaying package configuration, since apt-utils is not installed
#27 1.728 Selecting previously unselected package qpdf.
(Reading database ... 13019 files and directories currently installed.)
#27 1.761 Preparing to unpack .../qpdf/qpdf_10.6.3-1_amd64.deb ...
#27 1.783 Unpacking qpdf (10.6.3-1) ...
#27 1.965 Setting up qpdf (10.6.3-1) ...
#27 2.085 Installing pikepdf and dependencies
#27 2.086 + echo Installing pikepdf and dependencies
#27 2.086 + python3 -m pip install --no-cache-dir /pikepdf/usr/src/pikepdf/wheels/packaging*.whl
#27 2.425 WARNING: Requirement '/pikepdf/usr/src/pikepdf/wheels/packaging*.whl' looks like a filename, but the file does not exist
#27 2.425 ERROR: packaging*.whl is not a valid wheel filename.
#27 2.566 WARNING: You are using pip version 22.0.4; however, version 22.1.2 is available.
#27 2.566 You should consider upgrading via the '/usr/local/bin/python3 -m pip install --upgrade pip' command.
------
executor failed running [/bin/sh -c set -eux   && echo "Installing qpdf"     && apt-get install --yes --no-install-recommends /qpdf/usr/src/qpdf/libqpdf28_*.deb     && apt-get install --yes --no-install-recommends /qpdf/usr/src/qpdf/qpdf_*.deb   && echo "Installing pikepdf and dependencies"     && python3 -m pip install --no-cache-dir /pikepdf/usr/src/pikepdf/wheels/packaging*.whl     && python3 -m pip install --no-cache-dir /pikepdf/usr/src/pikepdf/wheels/lxml*.whl     && python3 -m pip install --no-cache-dir /pikepdf/usr/src/pikepdf/wheels/Pillow*.whl     && python3 -m pip install --no-cache-dir /pikepdf/usr/src/pikepdf/wheels/pyparsing*.whl     && python3 -m pip install --no-cache-dir /pikepdf/usr/src/pikepdf/wheels/pikepdf*.whl     && python -m pip list   && echo "Installing psycopg2"     && python3 -m pip install --no-cache-dir /psycopg2/usr/src/psycopg2/wheels/psycopg2*.whl     && python -m pip list]: exit code: 1

```

### Steps to reproduce

1. Run docker build: `COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1  docker build --build-arg JBIG2ENC_VERSION=0.29 --build-arg QPDF_VERSION=10.6.3 --build-arg PIKEPDF_VERSION=5.1.3 --build-arg PSYCOPG2_VERSION=2.9.3 .`
2. Wait and see

### Paperless-ngx version

main

### Host OS

Debian GNU/Linux 10 (buster)

### Installation method

Docker

**Comment by stumpylog** [comment]
This just needs the paths updated in the Dockerfile, following the change to use pip to build the wheels from a source dist

**Comment by Tockra** [comment]
@stumpylog okay I don't understand your answere. 
Feel free to fix the issue if you see the reason ;)

**Comment by qcasey** [comment]
I believe stumpylog means [this commit](https://github.com/paperless-ngx/paperless-ngx/commit/100f5422f655c9ecda8735b0ea32eac5ed223d2d), and that this will be fixed when `dev` is merged.

**Comment by stumpylog** [comment]
Yes, pretty much.  It's not ideal the building has broken, but it should be fully fixed with the next release.  And future updates to the installer images shouldn't be needed.

**Comment by shamoon** [comment]
Awesome, thanks for confirming. Let’s close this as it’s already resolved, we can always re-address as needed

should be releasing 1.8 beta soon!

**Comment by Tockra** [comment]
Is there any beta tag or similar which I can use until then?

**Comment by qcasey** [comment]
Try `dev`, our actions pipeline still builds so you should be fine there. The `beta` tag comes and goes.
